src/views/includes/confirmation-message.njk

```diff
@@ -0,0 +1,4 @@
+{% if props.data.subtype === "registered-office-address-change-check-your-answers" %}
```

Comment:
these 2 includes look to be doing the same thing
you should be able to keep the update-confirmation-message.njk and delete the confirmation-message.njk

Reply:
It's being strange not sure why it kept that, manually deleting.